docs/_src/tutorials/tutorials/1.md

```diff
@@ -237,9 +237,19 @@ prediction = pipe.run(
 
 
 ```python
+# Change `minimal` to `medium` or `all` to see a more detailed output
 print_answers(prediction, details="minimal")
 ```
 
+
+```python
+# Alternative: directly print the object
+
+# from pprint import pprint
+
+# pprint(prediction)
```

Comment:
I thought we also wanted to show the actual output here (at least the first answer in the json and then `...`) :thinking:

Reply:
Right! I put it in the .py script but somehow forgot to put it here :facepalm: